server/controllers/candidateOfficeController.js

```diff
@@ -0,0 +1,35 @@
+import dbConfig from '../database/dbConfig';
+
+class RegisterCandidate {
+  static register(req, res) {
+    const candidate = req.params.id;
+    const { office } = req.body;
+    console.log(req.body);
+    console.log(req.params.id);
+    dbConfig.query('INSERT INTO politico_andela.candidates (office, candidate) VALUES ($1, $2) RETURNING *', [office, candidate])
+      .then((candi) => {
```

Comment:
Use a better variable name as compared to `candi`

Reply:
Oh! that was used in hindsight... Changing it..